TurkishFilter/sections/general_url.txt

```diff
@@ -1,3 +1,10 @@
+!
+! This section contains the list of generic blocking rules.
```

Comment:
I've updated this in some other lists, could you please copy/paste it here?

Reply:
> ! This section contains the list of generic blocking rules.

I updated this text in all related files.